mesheryctl/internal/cli/root/system/start.go

```diff
@@ -301,6 +302,43 @@ func start() error {
 		if err != nil {
 			return err
 		}
+
+		operatorURL := "https://raw.githubusercontent.com/layer5io/meshery-operator/master/config/manifests/default.yaml"
```

Comment:
Thank you, @metonymic-smokey 🥇 

@navendu-pottekkat, these manifests should really be place into a central location.where do you suggest?

@hexxdump, should we update the Meshery bash installation script to include these manifests for Meshery Operator, which is essentially mandatory for running Meshery? @kumarabd, should these manifests be added to the artifacts for the brew and scoop packages?

@aneeshnema, are we caching manifest files locally? When `meshery system update` is invoked, should these Meshery Operator manifests be refreshed as well?

@Aisuko, these manifests should be accounted for in the air-gapped instructions.

Please create separate issues to track each of these considerations save for the first question of centralization, which can be addressed in this PR.

Reply:
@metonymic-smokey @leecalcote This could be placed here along with some related URLs.

https://github.com/layer5io/meshery/blob/bec7f0737a36ad1358d484ff749885eb1f30d4d7/mesheryctl/pkg/utils/helpers.go#L37

Reply:
Yes, mesheryctl does store the manifest files locally in `.meshery/manifests` folder.

So, from what I understand from what we are trying to do here, these manifest files should be updated and applied to the cluster.

Reply:
@leecalcote No these manifests are going to be dynamic, based on the operator version. So the current way seems better.

Reply:
So I'll undo my latest commit @kumarabd and keep it as is WRT URLs?

Reply:
@kumarabd, open an issue to have these manifests be added to the artifacts for the brew and scoop packages, please. List the issue number here.

Reply:
@leecalcote Wt I meant was to not add it to brew/scoop but continue pulling from the Meshery-operator repository. This is to maintain consistency.

Reply:
@leecalcote Wt I meant was to not add it to brew/scoop but continue pulling from the Meshery-operator repository. This is to maintain consistency.